apps/supernova/src/lib/queries/mutationFunctions.js

```diff
@@ -0,0 +1,6 @@
+import { deleteSilences, createSilences } from "./silencesQueries"
+
+export const MUTATION_FUNCTIONS = {
```

Comment:
I would suggest to name this function more specifric and domain specific since they are both Silences related and also put them not on lib queries lvl and more close to the places where they will be called.

Reply:
I mean there could also be other Mutations in the future and i wanted all mutations/queries at the dictionary kinda in one spot. So I don't have a better idea. And they are just called over the useBoundMutations Hook and never directly. You just mention the key which is looked up in this dict.

Reply:
Maybe I could rename vom "queries" dict to "api" and put it not in the lib subfolder?